src/components/Articles.jsx

```diff
@@ -0,0 +1,19 @@
+import { ArticleCard } from './ArticleCard';
+import { getArticles } from '../apiCalls';
+import { useEffect, useState } from 'react';
+
+export const Articles = () => {
+  const [articleList, setArticleList] = useState([]);
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticleList(articles);
+    });
+  }, []);
+  return (
+    <ul className='article-container'>
+      {articleList.map((article, i) => {
+        return <ArticleCard key={i} article={article} />;
+      })}
+    </ul>
```

Comment:
As discussed in slack, it would be ideal to have a loading pattern here. While the data from the api is being returned some information to the user would be great. While the user gets told what is going on, by forcing the render of the ul to wait you don't have problems trying to map over an empty array.